solutions/fscloud/version.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.3.0, <1.7.0"
+
```

Comment:
please leave a code comment to say `# Lock DA into an exact provider version - renovate automation will keep it updated`